### DISCUSSION TITLE
Hitsound samples between lazer and stable inconsistent

### DISCUSSION THREAD

**Discussion by minerplayer12** [ppy/osu#19546] [Q&A] [answered]
I booted up lazer with the skin I always use on stable, but the hitsounds are different between the two versions. osu!lazer uses the "clap" hitsound, while stable uses "normal" hitsounds.
https://user-images.githubusercontent.com/29819256/182317402-e794b211-557f-4615-8c34-0f673b1ba0de.mp4

Here is a .osk of the skin being used from my Google Drive (because it is too big for Github)
https://drive.google.com/file/d/1vq2WDg2f6tIfPKwWs_z0M-VtVTinBXda/view?usp=sharing
Edit: video wouldn't play so i changed it to one that would play

**Comment by peppy** [comment]
Could you please include some beatmaps where you are noticing this?

**Reply by minerplayer12** [reply]
It's every map that I play so I'm just wondering if it's just a me thing or if anyone else has this problem
https://osu.ppy.sh/beatmapsets/786141#osu/1744286
https://osu.ppy.sh/beatmapsets/448919#osu/1124153
https://osu.ppy.sh/beatmapsets/128931#osu/847314

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/pull/19574
